addons/mail/static/src/components/thread_viewer/thread_viewer_tests.js

```diff
@@ -476,6 +476,70 @@ QUnit.test('show message subject if thread is mailing channel', async function (
     )
 });
 
+QUnit.test('show only messages that need to be shown', async function (assert) {
```

Comment:
Name of test is too obscure. Better as "should not display user notification messages in non-mailboxes" or something like that.